===================================================================
RCS file: /cvsroot/ogle/ogle/configure.in,v
retrieving revision 1.126
retrieving revision 1.127
diff -u -r1.126 -r1.127
--- ogle/ogle/configure.in	2003/12/29 16:48:37	1.126
+++ ogle/ogle/configure.in	2003/12/30 14:09:06	1.127
@@ -495,7 +495,7 @@
           [ 
 #include <sys/asoundlib.h>
 #if defined(SND_LIB_MAJOR) && defined(SND_LIB_MINOR)
-#if SND_LIB_MAJOR>0 || (SND_LIB_MAJOR==0 && SND_LIB_MINOR==5)
+#if SND_LIB_MAJOR==0 && SND_LIB_MINOR<6
 AP_maGiC_VALUE
 #endif
 #endif
